lib/validation/rules/url.ts

```diff
@@ -1,8 +1,14 @@
-import { url as baseUrl, helpers } from '@vuelidate/validators'
+import { createRule } from '../Rule'
+import { url as baseUrl } from '../validators'
+
+export const message = {
+  en: 'The value is invalid.',
```

Comment:
Let's match the message with `email` rule. "The URL is invalid."

Reply:
Fixed: e5c53aa